projects/nordic/nrf52840-dk/ses/aws_demos/aws_demos.emProject

```diff
@@ -589,6 +582,7 @@
             <file file_name="../../../../../vendors/nordic/nRF5_SDK_15.2.0/modules/nrfx/mdk/ses_startup_nrf_common.s" />
             <file file_name="../../../../../vendors/nordic/nRF5_SDK_15.2.0/modules/nrfx/mdk/ses_startup_nrf52840.s" />
             <file file_name="../../../../../vendors/nordic/nRF5_SDK_15.2.0/modules/nrfx/mdk/system_nrf52.c" />
+            <file file_name="../../../../../vendors/nordic/nRF5_SDK_15.2.0/modules/nrfx/mdk/nrf52840.h" />
```

Comment:
Are we introducing additional dependency here?

Reply:
The include file is actually being used and the path is already part of the include paths. It's just that file was not added to project.